dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/lksg/LksgFramework.kt

```diff
@@ -0,0 +1,114 @@
+package org.dataland.frameworktoolbox.frameworks.lksg
+
+import org.dataland.frameworktoolbox.frameworks.InDevelopmentPavedRoadFramework
+import org.dataland.frameworktoolbox.intermediate.Framework
+import org.dataland.frameworktoolbox.intermediate.components.MultiSelectComponent
+import org.dataland.frameworktoolbox.intermediate.components.SingleSelectComponent
+import org.dataland.frameworktoolbox.intermediate.components.support.SelectionOption
+import org.dataland.frameworktoolbox.intermediate.group.ComponentGroup
+import org.dataland.frameworktoolbox.intermediate.group.ComponentGroupApi
+import org.dataland.frameworktoolbox.intermediate.group.edit
+import org.dataland.frameworktoolbox.intermediate.group.getOrNull
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.LabelBadgeColor
+import org.springframework.stereotype.Component
+import java.io.File
+
+/**
+ * The Lksg Framework
+ */
+@Component
+class LksgFramework : InDevelopmentPavedRoadFramework(
```

Comment:
should not be indev in the final version

Reply:
this currently cant be changed to PavedRoadFramework, because the upload config generates duplicate import statements, that can only be removed by hand.
If @MarcTM01 enables typescript imports for the uploadConfig, this bug will be gone.

Reply:
Do you know why these duplicate statements are generated?